.github/workflows/build-and-test-pkg.yaml

```diff
@@ -58,7 +58,10 @@ jobs:
           git clean -f -d
           make clean
           make download-licenses 
-          make FINCH_OS_IMAGE_LOCATION_ROOT=/Applications/Finch
+          CGO_CFLAGS="-mmacosx-version-min=11.0" \
+            CGO_LDFLAGS="-mmacosx-version-min=11.0" \
+            FINCH_OS_IMAGE_LOCATION_ROOT=/Applications/Finch \
+            make
```

Comment:
Should we consider pushing this down into the Makefile so it is easier to update the min version if required?

Reply:
Yeah, that's a good point. Do you think it makes sense to have a new target just for releases, or to just always set these parameters?

Reply:
I just added it for all mac targets, PTAL

Reply:
> I just added it for all mac targets, PTAL

Should be fine sense our docs state a minimum anyways. Advanced users can always override with the environment variable.